chaos_genius/views/kpi_view.py

```diff
@@ -41,6 +41,7 @@
 from chaos_genius.settings import SUMMARY_DEEPDRILLS_ENABLED_TIME_RANGES
 from chaos_genius.utils.pagination import pagination_args, pagination_info
 from chaos_genius.utils.search import SEARCH_PARAM_NAME, make_search_filter
+from chaos_genius.utils.utils import make_path_safe
```

Comment:
Do we need this?

Reply:
No, it's not needed. Got left over from another change.